frontend/genebanks.tsx

```diff
@@ -1,135 +1,66 @@
-  breadcrumb: {
-    textDecoration: "none",
-  },
-  spacer: {
-    height: '39px',
-  },
-  table: {
-    height: "calc(100% - 75px)",
-    padding: "0 20px",
-    overflowY: "scroll",
+  buttonBar: {
+    padding: "5px",
   },
 });
 
 /**
- * Provides genebanks management forms for granting and revoking herd
- * permissions, and managing herd animals.
+ * Provides buttons to select which genebank to view, which is then done with
+ * the GenebankView function.
  */
 export function Genebanks() {
+  const styles = useStyles();
+  const {url} = useRouteMatch()
+  const history = useHistory()
   const {genebanks} = useDataContext()
+  const [subpath, setSubpath] = React.useState('' as String)
   const [genebank, setGenebank] = useState(undefined as Genebank | undefined)
-  const [individuals, setIndividuals] = React.useState([] as Array<Individual>)
-  const styles = useStyles();
 
   React.useEffect(() => {
-    if (genebanks.length > 0 && !genebank) {
-      selectGenebank(genebanks[0].id);
-    }
-  }, [genebanks]);
+    setSubpath(location.pathname.replace(url, '').trim().replace(/\//, ''))
+  })
 
-  const selectGenebank = (id: number) => {
-    if (!genebanks) {
-      return;
-    }
-    const targetGenebank = genebanks.find((g: Genebank) => g.id == id);
-    if (!targetGenebank) {
-      return;
+  React.useEffect(() => {
+    if (!subpath && genebanks.length > 0) {
+      history.push(`${url}/${genebanks[0].name}`)
+    } else if (genebanks.length > 0) {
+      const targetGenebank = genebanks.find((g: Genebank) => g.name.toLowerCase() == subpath.toLowerCase())
+      if (targetGenebank) {
+        setGenebank(targetGenebank)
+      }
     }
-    setGenebank(targetGenebank);
-    setIndividuals(targetGenebank.individuals);
-  }
+  }, [subpath, genebanks]);
 
   return <>
     <Paper>
-      {genebanks.length > 1 && genebanks.map((g: Genebank, i: number) => {
-          return <Button  key={g.id}
-                          variant='contained'
-                          value={g.id}
-                          color={genebank && genebank.id == g.id ? 'primary' : 'default'}
-                          onClick={(e: any) => selectGenebank(e.currentTarget.value)}>{g.name}</Button>
-        })
-      }
-
-      <div className={styles.table}>
-        <MaterialTable
-          icons={tableIcons}
-          columns={columns}
-          data={individuals}
-          title="Alla individer"
-        />
+      <div className={styles.buttonBar}>
+        {genebanks.length > 1 && genebanks.map((g: Genebank, i: number) => {
+            return <Link to={`${url}/${g.name}`} key={g.id}>
+                <Button variant='contained'
+                        value={g.id}
+                        color={genebank && genebank.id == g.id ? 'primary' : 'default'}
+                        >
+                  {g.name}
+                </Button>
+              </Link>
+          })
+        }
       </div>
+      { genebank && <GenebankView genebank={genebank} /> }
```

Comment:
Right now `GenebankView` can be unnecessarily redrawn with the same genebank. This can be fixed by either wrapping the `GenebankView` component in `React.memo` or using `React.useMemo` here:
```typescript
      { React.useMemo(() => genebank && <GenebankView genebank={genebank} />, [genebank]) }
```

Reply:
nice! I'll use that! :)